src/plugin/customParseFormat/index.js

```diff
@@ -9,7 +9,7 @@ const matchUpperCaseAMPM = /[AP]M/
 const matchLowerCaseAMPM = /[ap]m/
 const matchSigned = /[+-]?\d+/ // -inf - inf
 const matchOffset = /[+-]\d\d:?\d\d/ // +00:00 -00:00 +0000 or -0000
-const matchWord = /\d*[^\s\d]+/ // Word
+const matchWord = /\d*[^\s\d-:/.()]+/ // Word
```

Comment:
Good fix.

Do we need to list more possible divider (such as \ / _ etc, I'm not sure)?

Reply:
Yes, we can include if we are considering them as the format separator, in this case, we also need to modify https://github.com/iamkun/dayjs/blob/9b3ba887fe1428e4a963daa9b691e2f2db36cb1d/src/plugin/customParseFormat/index.js#L1.

Should I include these dividers("_" and "\")?

Reply:
Not sure, to be honest.

Maybe we could add these later but not this time. 😬

Reply:
Thanks @iamkun. :)